### PR TITLE
Add LogWithName attribute

### PR DESCRIPTION
**Pull request by paymanx**
The usage of this attribute is when you want to save properties with different name.

When you want to log object properties with different name you can use this attribute.
For example, you have a class that has a MessageType property but you want to log MessageType data with "mt" to reduce log file size.

You can use:

```
class Message
{
    [LogWithName("mt")]
    public int MessageType
    {
        get;
        set;
    }
}
```

### CODE REVIEW

File: test/Destructurama.Attributed.Tests/LogWithNameAttributedTests.cs [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    public class UserAuthData
+    {
+        public string Username { get; set; }
+
+        [NotLogged]
+        public string Password { get; set; }
+    }
+
+    [TestFixture]
+    public class AttributedDestructuringTests
```
Comment (sungam3r) [inline test/Destructurama.Attributed.Tests/LogWithNameAttributedTests.cs:46]: You messed up the filenames.

File: appveyor.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 version: '{build}'
 skip_tags: true
-image: Visual Studio 2019
+image: Visual Studio 2017
```
Comment (sungam3r) [inline appveyor.yml:3]: ?

File: test/Destructurama.Attributed.Tests/LogWithNameAttributedTests.cs (hunk 1 above)
Comment (paymanx) [reply test/Destructurama.Attributed.Tests/LogWithNameAttributedTests.cs]: fixed

File: appveyor.yml (hunk 2 above)
Comment (paymanx) [reply appveyor.yml]: fixed

**Comment by paymanx** [comment]
the issues has been resovled

**Pull request merged**